plugins/modules/one_image.py

```diff
@@ -24,23 +24,6 @@
   diff_mode:
     support: none
 options:
-  api_url:
```

Comment:
You are **removing** module options here. That's a breaking change and thus not allowed.

Reply:
No, you can still use them, they are now in module_utils/opennebula, much like in the following modules:
https://github.com/ansible-collections/community.general/blob/main/plugins/modules/one_template.py
https://github.com/ansible-collections/community.general/blob/main/plugins/modules/one_vnet.py